x/burn/module.go

```diff
@@ -0,0 +1,118 @@
+package burn
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/medibloc/panacea-core/x/burn/internal/keeper"
+
+	"github.com/cosmos/cosmos-sdk/x/supply"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+	"github.com/gorilla/mux"
+	"github.com/spf13/cobra"
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	"github.com/medibloc/panacea-core/x/burn/internal/types"
+)
+
+type AppModuleBasic struct{}
+
+type AppModule struct {
+	AppModuleBasic
+	keeper       keeper.Keeper
+	supplyKeeper supply.Keeper
```

Comment:
It seems this `supplyKeeper` is not needed, because the `burn.Keeper` already contains a `supplyKeeper`.

Reply:
@youngjoon-lee 
That's right. I deleted it!